node/cloud-functions/README.md

```diff
@@ -0,0 +1,181 @@
+- [Create a Bigtable with a column family using CBT](#create-a-bigtable-with-a-column-family-using-cbt)
+  - [Writing a Cloud Function](#writing-a-cloud-function)
+  - [Deploying the Cloud Function](#deploying-the-cloud-function)
+- [Triggering the Cloud Function](#triggering-the-cloud-function)
+- [Verifying everything worked](#verifying-everything-worked)
+- [Cleaning up](#cleaning-up)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Before you begin
+
+This sample assumes you've finished the [Hello World][hello-world] example. If
+you haven't, you should complete the same steps outlined in the Hello World
+example to get set up.
+
+Be sure to checkout the
+[Cloud Functions Quickstart Guide][cloud-functions-quickstarts]. The quickstarts
+come in [CLI][cloud-functions-quickstart-cli] and
+[console][cloud-functions-quickstart-console] flavors.
+
+[cloud-functions-quickstarts]: https://cloud.google.com/functions/docs/quickstarts
+[cloud-functions-quickstart-cli]: https://cloud.google.com/functions/docs/quickstart
+[cloud-functions-quickstart-console]: https://cloud.google.com/functions/docs/quickstart-console
+
+
+### Enabling the Cloud Functions API
+
+Make sure to [enable the Cloud Functions API][enable-cloud-functions].
+
+[enable-cloud-functions]: https://console.cloud.google.com/flows/enableapi?apiid=cloudfunctions
+
+[hello-world]: ../hello-world
+
+### Set up a cloud storage bucket
+
+If you haven't already done so in the Cloud Functions Quickstart, set up a 
+Cloud Storage bucket replacing `PROJECT_ID` with the project ID and 
+`BUCKET_NAME` with some unique bucket name:
+
+    gsutil mb -p PROJECT_ID gs://BUCKET_NAME
+
+### Installing CBT
+
+This example will make use of the [cbt][cbt] tool, it can be installed by
+running the following:
+
+    gcloud components update
+    gcloud components install cbt
+
+[cbt]: https://cloud.google.com/bigtable/docs/go/cbt-overview
+
+## Create a Bigtable with a column family using CBT
+
+The first thing needed is to create a Bigtable with a column family. The
+Bigtable will be named `cloud-functions` and the column family will be `cf1`:
+
+    cbt createtable cloud-functions
+    cbt createfamily cloud-functions cf1
+
+### Writing a Cloud Function
+
+The Cloud Function in this example will have Pub/Sub topic named
+`hello_bigtable`. Look at [`index.js`][index-js] to see how the Cloud Function
+works. Notice the dependencies used are included in
+[`package.json`][package-json].
+
+[index-js]: index.js
+[package-json]: package.json
+
+### Deploying the Cloud Function
+
+When deploying a Cloud Function, make sure you're in the directory that
+contains the `index.js` file you want to deploy:
+
+    cd cloud-functions
+
+Run the following command to deploy the Cloud Function (make sure to replace
+`BUCKET_NAME` with what you used for the bucket name):
+
+    gcloud beta functions deploy helloBigtable --stage-bucket BUCKET_NAME --trigger-topic hello_bigtable
```

Comment:
Can you please add --region here, and specify that it should be in the same region as the Cloud Bigtable cluster?

Reply:
Done